frameit/lib/frameit/options.rb

```diff
@@ -9,104 +9,111 @@
-                                     type: Boolean),
-        FastlaneCore::ConfigItem.new(key: :force_orientation_block,
-                                       type: :string_callback,
-                                       description: "[Advanced] A block to customize your screenshots' device orientation",
-                                       display_in_shell: false,
-                                       optional: true,
-                                       default_value: proc do |filename|
-                                         f = filename.downcase
-                                         if f.end_with?("force_landscapeleft")
-                                           :landscape_left
-                                         elsif f.end_with?("force_landscaperight")
-                                           :landscape_right
-                                         end
-                                       end,
-                                       default_value_dynamic: true),
-        FastlaneCore::ConfigItem.new(key: :debug_mode,
-                                       env_name: "FRAMEIT_DEBUG_MODE",
-                                       description: "Output debug information in framed screenshots",
-                                       default_value: false,
-                                       type: Boolean),
-        FastlaneCore::ConfigItem.new(key: :resume,
-                                       env_name: "FRAMEIT_RESUME",
-                                       description: "Resume frameit instead of reprocessing all screenshots",
-                                       default_value: false,
-                                       type: Boolean),
-        FastlaneCore::ConfigItem.new(key: :use_platform,
-                                       env_name: "FRAMEIT_USE_PLATFORM",
-                                       description: "Choose a platform, the valid options are IOS, ANDROID and ANY (default is either general platform defined in the fastfile or IOS to ensure backward compatibility)",
-                                       optional: true,
-                                       default_value: Platform.symbol_to_constant(Fastlane::Helper::LaneHelper.current_platform),
-                                       default_value_dynamic: true,
-                                       verify_block: proc do |value|
-                                         UI.user_error!("Invalid platform type '#{value}'. Available values are " + Platform.all_platforms.join(', ') + ".") unless ConfigParser.supported_platform?(value)
-                                       end)
-      ]
+        options = color_options.map do |option|
+          FastlaneCore::ConfigItem.new(
+            key: option[:key],
+            env_name: option[:env_name],
+            description: option[:description],
+            type: Boolean,
+            default_value: false,
+            optional: true
```

Comment:
```suggestion
```

We don't need this, since it's a boolean and it defaults to false already :) we'd only want to make it optional if explicitly passing `nil` was a thing

Reply:
done